src/parser/shared/modules/items/bfa/raids/bod/IncandescentSliver.js

```diff
@@ -0,0 +1,100 @@
+import ItemIcon from 'common/ItemIcon';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+
+
+class IncandescentSliver extends Analyzer {
+  critRating = 0;
+  masteryRating = 0;
+
+  static dependencies = {
+    statTracker: StatTracker,
+    combatants: Combatants,
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrinket(ITEMS.INCANDESCENT_SLIVER.id);
+    if(this.active) {
+      const item = this.selectedCombatant.getItem(ITEMS.INCANDESCENT_SLIVER.id);
+      this.critRating = calculateSecondaryStatDefault(385, 20, item.itemLevel);
+      this.masteryRating = calculateSecondaryStatDefault(385, 66, item.itemLevel);
+      this.statTracker.add(SPELLS.INCANDESCENT_LUSTER.id, {
+        crit: this.critRating,
+      });
+      this.statTracker.add(SPELLS.INCANDESCENT_BRILLIANCE.id, {
+        mastery: this.masteryRating,
+      });
+    }
+  }
+
+  averageCritRating() {
+    const averageStacks = this.selectedCombatant.getStackWeightedBuffUptime(SPELLS.INCANDESCENT_LUSTER.id) / this.owner.fightDuration;
+    return averageStacks * this.critRating;
+  }
+
+  averageMasteryRating() {
+    return this.masteryRating * this.masteryUptime;
+  }
+
+  get critUptime() {
+    return this.selectedCombatant.getBuffUptime(SPELLS.INCANDESCENT_LUSTER.id)/this.owner.fightDuration;
+  }
+
+  get masteryUptime() {
+    return this.selectedCombatant.getBuffUptime(SPELLS.INCANDESCENT_BRILLIANCE.id)/this.owner.fightDuration;
+  }
+
+  item() {
+    const playersWithTrinket = [];
+    Object.keys(this.combatants.players).forEach((key) => {
+      const player = this.combatants.players[key];
+      if(player.hasTrinket(ITEMS.INCANDESCENT_SLIVER.id) && player._combatantInfo.name !== this.selectedCombatant._combatantInfo.name){
+        playersWithTrinket.push(player);
+      }
+    });
+    const playersTable = (
+      <table className="table table-condensed">
+        <thead>
+          <tr>
+            <th>Other Players With Trinket</th>
+          </tr>
+        </thead>
+        <tbody>
+          {
+            playersWithTrinket.map((player) => {
+              return (
+                <tr key={player._combatantInfo.name}>
+                  <th><p className={player.spec.className}>{player._combatantInfo.name}</p></th>
+                </tr>
+              );
+            })
+          }
+        </tbody>
+      </table>
+    );
+    return (
+      <StatisticBox
+        icon={<ItemIcon id={ITEMS.INCANDESCENT_SLIVER.id} />}
+        value={`${formatNumber(this.averageCritRating())} Average Crit, ${formatNumber(this.averageMasteryRating())} Average Mastery`}
+        label={`Incandescent Sliver`}
+        tooltip={`Your crit buff had an uptime of <b>${formatPercentage(this.critUptime)}%</b> and your mastery buff had an uptime of <b>${formatPercentage(this.masteryUptime)}%</b>`}
+        category={STATISTIC_CATEGORY.ITEMS}>
+        {playersTable}
```

Comment:
I think these should be moved to the tooltip. It's not critical information to have, so I feel it shouldn't attract as much attention as an expandable statistic attracts.

Reply:
I moved the player information into the tooltip rather than the expandable table. Since the tooltip prop has to be a string though I don't believe that we can get the class colours like you wanted in the original issue.

![image](https://user-images.githubusercontent.com/6363390/52093400-0f5cd480-2589-11e9-97db-00327567fc7f.png)

Reply:
We'll get that ability in next. This is excellent for now. Thanks :)